ImAMWorkaround/README.md

```diff
@@ -0,0 +1,25 @@
+# Example Template for Creating WhaleSongs Walkthroughs
```

Comment:
Please remove this file and directory as it does not match the pattern for this repository.